src/tests/ftest/util/daos_core_base.py

```diff
@@ -179,10 +165,41 @@ def run_subtest(self):
                 self.fail(
                     "{0} failed with return code={1}.\n".format(
                         job_str, result.result.exit_status))
+        finally:
+            # List any remote cmocka files
+            self.log.debug("Remote %s directories:", cmocka_dir)
+            ls_command = "ls -al {}".format(cmocka_dir)
+            clush_ls_command = "{} {}".format(
+                get_clush_command(self.hostlist_clients, "-B -S"), ls_command)
+            log_task(self.hostlist_clients, clush_ls_command)
+
+            self.log.debug("Local %s directory before clush:", cmocka_dir)
+            run_command(ls_command)
+
+            # Copy any remote cmocka files back to this host
+            for cmocka_file in os.listdir(cmocka_dir):
```

Comment:
The `os.listdir(cmocka_dir)` will only list the cmocka files on the launch node.  This will work for the suite.py, rebuild.py, and nvme_recovery.py tests (because the test runs on the launch node), but it will not list the remote cmocka files generated when the dfs.py test is run.